### PR TITLE
Devdocs-3314 - update links to csv

### PR DESCRIPTION
**Pull request by markcmurphy**
As part of adding Mainfreight to CSV, updated links to point to Github repo instead of Google Sheets file.

### CODE REVIEW

File: reference/orders.v2.oas2.yml [hunk 1]
```diff
@@ -3624,7 +3625,7 @@ components:
           title: Tracking Carrier
           description: |-
             Tracking carrier for the shipment.
-            Acceptable values include an empty string (`""`) or one of the valid tracking-carrier values viewable [here](https://github.com/bigcommerce/dev-docs/blob/development/assets/csv/tracking_carrier_values.csv) and downloadable as a .CSV file [here](https://raw.githubusercontent.com/bigcommerce/dev-docs/development/assets/csv/tracking_carrier_values.csv).
+            Acceptable values include an empty string (`""`) or one of the valid tracking-carrier values viewable and downloadable as a .CSV file [here](https://github.com/bigcommerce/dev-docs/blob/master/assets/csv/tracking_carrier_values.csv).
```
Comment (bc-andreadao) [inline reference/orders.v2.oas2.yml:3628]:
```suggestion
            Acceptable values include an empty string (`""`) or one of the valid tracking-carrier values that you can view and download as a .CSV file [here](https://github.com/bigcommerce/dev-docs/blob/master/assets/csv/tracking_carrier_values.csv).
```

File: reference/orders.v2.oas2.yml [hunk 2]
```diff
@@ -4002,7 +4002,7 @@ components:
           title: Tracking Carrier
           description: |-
             Tracking carrier for the shipment.
-            Acceptable values include an empty string (`""`) or one of the valid tracking-carrier values viewable [here](https://docs.google.com/spreadsheets/d/1w9c_aECSCGyf-oOrvGeUniDl-ARGKemfZl0qSsav8D4/pubhtml?gid=0&single=true) and downloadable as a .CSV file [here](https://docs.google.com/spreadsheets/d/1mTueEynfcEmwsU2y2Jd2MX-8GKwNZrmlRMBcIElg9aY/pub?gid=0&single=true&output=csv).
+            Acceptable values include an empty string (`""`) or one of the valid tracking-carrier values viewable and downloadable as a .CSV file [here](https://github.com/bigcommerce/dev-docs/blob/master/assets/csv/tracking_carrier_values.csv).
```
Comment (bc-andreadao) [inline reference/orders.v2.oas2.yml:4005]:
```suggestion
            Acceptable values include an empty string (`""`) or one of the valid tracking-carrier values that you can view and download as a .CSV file [here](https://github.com/bigcommerce/dev-docs/blob/master/assets/csv/tracking_carrier_values.csv).
```

**Review by bc-andreadao — APPROVED** [review]
Minor suggestion for wording

File: reference/orders.v2.oas2.yml (hunk 2 above)
Comment (markcmurphy) [reply reference/orders.v2.oas2.yml:4005]: Good suggestion!

**Pull request merged**